docs/digger/installation/macos-prereqs.asciidoc

```diff
@@ -51,13 +142,16 @@ The machine requires a network access to the following hosts:
 | https://raw.githubusercontent.com
 | To download and install NVM.
 
-| https://rvm.io
-| To download and install RVM.
-
 | http://developer.apple.com
 | To download developer certificates and Xcode.
 
 | https://npmjs.org
 | To install NPM packages.
+
+| https://updates.jenkins-ci.org/download/plugins/*
+| To install the required Jenkins plugins via the update centre
```

Comment:
I think we are not installing jenkins plugins on macos, so we don't need this ?

Reply:
Correct, removing